caneton/decode.py

```diff
@@ -155,13 +154,14 @@ def message_decode(message_id, message_length, message_data, dbc_json):
         message_binary_length)
 
     if message_info.get('has_multiplexor', False):
-        multiplexing_mode = message_get_current_multiplexing_mode(
+        multiplexor = message_get_multiplexor(
```

Comment:
Could the function be replaced by  `multiplexor = int(message_data[:8])`?
From what I see, only signals have an _endianess_ so the messages must have the multiplexor always in the same bits.